src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltype/InputIllegalTypeSameFileName.java

```diff
@@ -27,6 +27,6 @@ private void foo() {
     java.util.List<Integer> list = new ArrayList<>(); //WARNING
     private ArrayList<String> values;
     private Boolean d; //WARNING
-    private Boolean[] d1; //WARNING
+    private Boolean[] d1;
     private Boolean[][] d2; //WARNING
```

Comment:
please remove "//WARNING" on this line. As you removed expected message in UT method.

Reply:
But now there is expected violation on that line for new test (IllegalTypeCheckTest.testPlainAndArrayTypes)

Reply:
Please split Input files. If in Input file declared configuration and trailing comments , they should stay actual all the time.

Reply:
done